app/views/submissions/submit_comment.erb

```diff
@@ -1,17 +1,16 @@
 <% if current_user.guest? %>
-  <p> You're not logged in right now.Please login via <a href="/login">GitHub </a> to comment </p>
+  <p> You're not logged in right now. Please login via <a href="/login">GitHub </a> to comment </p>
```

Comment:
Yes. If the user is redirected to dashboard. He will have to start from stream again. Returning him to same submission page is good . I will look into it. Thank you.